GDJS/Runtime/howler-sound-manager/howler-sound-manager.ts

```diff
@@ -746,6 +757,29 @@ namespace gdjs {
       return this._musics[channel] || null;
     }
 
+    setSoundSpatialPositionOnChannel(
+      channel: number,
+      x: number,
+      y: number,
+      z: number
+    ) {
+      const sound = this.getSoundOnChannel(channel);
+      if (sound && !sound.paused()) sound.setSpatialPosition(x, y, z);
+      else {
+        // If no sound is playing at the time the method is called, the
+        // position is cached and will be used by the `playSoundOnChannel` method
+        // to set the spatial position right after the sound starts playing.
+        // This cached value is then cleared at the end of the frame.
+        // Without this caching strategy, if actions are in the wrong order,
+        // the spatial position is set a frame after the sound starts playing.
```

Comment:
should the last part be more "if actions are in the wrong order, the spatial position will not apply to the sound that hasn't started playing yet"